### DISCUSSION TITLE
github

### DISCUSSION THREAD

**Discussion by George-Nascimento** [community/community#124428] [New to GitHub]
### Select Topic Area

Question

### Body

Sou novo neste lugar. Me disseram que é uma rede social para programadores...?

**Comment by davevad93** [comment]
Hi @George-Nascimento and welcome to GitHub,

This platform is used mainly for storing, tracking, and collaborating on open-source projects. It can also serve as a social network to connect with other developers but its main purpose is hosting code that allows for version control and collaboration. 

If you have any doubts or questions feel free to ask here in the Community discussions, there are plenty of knowledgeable people willing to help!

**Comment by LiteBrite82** [comment]
Welcome to GitHub and the GitHub Community @George-Nascimento  🎆 ! We're glad you're here and can't wait to see where your GitHub journey takes you. Check out this [check-in](https://github.com/orgs/community/discussions/42302#discussion-4683140) with links on how to choose your first project to contribute to, a blog with getting started resources, and a new user checklist.

Best of luck!

**Comment by Carlos-Levi** [comment]
Também é brasileiro? acredito que sim 😄. Se precisar de ajuda é só falar!
